ui/app/[locale]/(authorized)/kafka/[kafkaId]/consumer-groups/[groupId]/ResetOffsetModal.tsx

```diff
@@ -69,11 +70,13 @@ export function ResetOffsetModal({
         <StackItem>
           <Text>{t("shutdown_active_members")}</Text>
         </StackItem>
-        <StackItem>
-          <ExternalLink testId={"learn_to_shutdown_members"} href={""}>
-            {t("learn_to_shutdown_members")}
-          </ExternalLink>
-        </StackItem>
+        {isProductizedBuild && (
+          <StackItem>
+            <ExternalLink testId={"learn_to_shutdown_members"} href={"https://docs.redhat.com/en/documentation/red_hat_streams_for_apache_kafka/2.7/html-single/kafka_configuration_tuning/index#managing_offset_policy"}>
+              {t("learn_to_shutdown_members")}
+            </ExternalLink>
```

Comment:
The label and link should maybe go under `learning.labels` and `learning.links`. We should also have a link to something in the Apache Kafka documentation for upstream.

Reply:
@MikeEdgar Is the phrase "Learn how to shut down members" correct to include under learning resources?

Reply:
@hemahg , yes that seems good.

Reply:
Isn't the wording a bit... aggressive? Not a native speaker here, but it feels that way

Reply:
@shirimordechay, We've been discussing the phrasing for the learning resource, and I wanted to get some additional feedback. @MikeEdgar suggested "Learn how to stop consumers" as a more appropriate phrase, but I'd like to make sure everyone is on board with this change. Do you have any thoughts or suggestions for improving the wording or can help find relevant documentation links?

Reply:
Rephrasing sounds good to me. I believe the doc link is the one @pmellor@redhat.com has provided, but I'm assuming we'll have a new link for GA. need to confirm that with Paul.

Reply:
Update the phrase as per @MikeEdgar suggestion, there is no specific document to shutdown the consumer group members